index.html

```diff
@@ -53,5 +53,158 @@ <h2 class="job">I'm a software developer</h2>
         <a href="#"><img src="./images/Disabled.svg" alt="next_button" /></a>
       </div>
     </section>
+    <section class="works">
+      <a class="title" href="#">My Recent Works</a>
+      <hr />
+      <div class="cards">
+        <div class="card">
+          <img
+            class="img_placeholder"
+            src="./images/Im_Plaholder2.png"
+            alt="Img_Plaholder"
+          />
+          <div class="project_info">
+            <h4 class="project_name">Multi-Post Stories Gain+Glory</h4>
+            <ul class="languages_used">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
+            <a class="see_project" href="#">See Project</a>
+          </div>
+        </div>
+        <div class="card">
+          <img
+            class="img_placeholder"
+            src="./images/Im_Plaholder2.png"
+            alt="Img_Plaholder"
+          />
+          <div class="project_info">
+            <h4 class="project_name">Multi-Post Stories Gain+Glory</h4>
+            <ul class="languages_used">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
+            <a class="see_project" href="#">See Project</a>
+          </div>
+        </div>
+        <div class="card">
+          <img
+            class="img_placeholder"
+            src="./images/Im_Plaholder2.png"
+            alt="Img_Plaholder"
+          />
+          <div class="project_info">
+            <h4 class="project_name">Multi-Post Stories Gain+Glory</h4>
+            <ul class="languages_used">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
```

Comment:
- [ ] Nice to use the list on there. Please fix the typo. Instead of `JavScript` you should write `JavaScript.` You should change all of them on the page.
_I see it is JavScript in design. But it is not correct._